app/models/miq_ae_class.rb

```diff
@@ -29,7 +29,7 @@ def self.lookup_by_namespace_and_name(name_space, name, _args = {})
     name_space = MiqAeNamespace.lookup_by_fqname(name_space)
     return nil if name_space.nil?
 
-    find_by(:namespace_id => name_space.id).where(arel_table[:name].lower.eq(name.downcase))
+    where(:namespace_id => name_space.id).find_by(arel_table[:name].lower.eq(name.downcase))
```

Comment:
Isn't this the same change as in https://github.com/ManageIQ/manageiq/pull/20044 which has been merged?

Reply:
:+1: thanks